### ISSUE TITLE
State 'LAUNCHING_PENDING' raises an attribute error

### ISSUE THREAD

**Issue by mturilli** [radical-cybertools/radical.analytics#13]
RP: experiment/aimes
SID: rp.session.radical.mingtha.017035.0010
```
pp(session.describe('state_values', etype=['pilot']))
{'pilot': {'state_values': {-1: None,
                            0: 'NEW',
                            1: 'LAUNCHING_PENDING',
                            2: 'LAUNCHING',
                            3: 'ACTIVE_PENDING',
                            4: 'ACTIVE',
                            5: ['FAILED', 'DONE', 'CANCELED']}}}
pilots = session.filter(etype='pilot', inplace=False)
pilots.duration([rp.LAUNCHING_PENDING, rp.LAUNCHING])

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'LAUNCHING_PENDING'
```

**Comment by vivek-bala** [comment]
Maybe ```rp.PENDING_LAUNCH``` ?

Devel has https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/states.py#L70

**Comment by andre-merzky** [comment]
Currently, the pilot states are defined as:

```python
PMGR_LAUNCHING_PENDING = 'LAUNCHING_PENDING'
PMGR_LAUNCHING         = 'LAUNCHING'
PMGR_ACTIVE_PENDING    = 'ACTIVE_PENDING'
PMGR_ACTIVE            = 'ACTIVE'
```

so the strings don't have the `PMGR_` prefix.  I can see how that is inconsistent -- I'll add the prefixes to the strings on the RP layer.  Either way, `rp.PMGR_LAUNCHING_PENDING` should work as expected.

**Comment by andre-merzky** [comment]
Well, before changing the state names, maybe I'll wait for your opinion on that :)

**Comment by andre-merzky** [comment]
We agreed to add the prefix for being more informative.

**Comment by andre-merzky** [comment]
This is now implemented in the RP analytics branch 3b39e4a

**Comment by andre-merzky** [comment]
I should mention that this now breaks the analysis of old profiles, since the state strings in those profiles changes.  Since the profiles are text based, the following script will fix them:

```sh
#!/bin/sh

for f in $*
do
    echo "convert $f"
    sed -i 's/\bLAUNCHING_PENDING\b/PMGR_LAUNCHING_PENDING/g' "$f"
    sed -i 's/\bLAUNCHING\b/PMGR_LAUNCHING/g' "$f"
    sed -i 's/\bACTIVE_PENDING\b/PMGR_ACTIVE_PENDING/g' "$f"
    sed -i 's/\bACTIVE\b/PMGR_ACTIVE/g' "$f"
done
```
